### ISSUE TITLE
discrete depth instead of continous depth

### ISSUE THREAD

**Issue by qsisi** [YvanYin/Metric3D#199]
Hello!

I tried to train the ConvNeXt-Tiny model solely on VKITTI2 without loading any pre-trained model, after some steps, here are the results compared to the released convnext-tiny:
![image](https://github.com/user-attachments/assets/5519d0a7-2849-4fc0-930f-f5858cff152f)
more interestingly, the point cloud reconstructed by my trained model is much more "discrete" compared to the model you released:
![image](https://github.com/user-attachments/assets/7485b560-c42b-4309-93c0-ee486be8fcd1)
It seems to me something goes wrong **when upsampling the 1/4 depth map to full-resolution depth map.** 
I think the only difference between my trained model and your released model is that the released model was trained on a larger mixed dataset, while my trained model is only trained on VKITTI2. Have you encounter such problems? or should I just use more data in training can solve this problem? or the learning rate I used is different from yours.. etc...

Thank you very much for your help.
